Source/Fuse.Common/Timer.uno

```diff
@@ -54,10 +60,11 @@ namespace Fuse
 			method can not be relied upon for very short durations. For high-performance timing,
 			consider spawning a new thread. 
 		*/
-		public static void Wait(double duration, Action callback)
+		public static Timer Wait(double duration, Action callback)
 		{
 			var t = new Timer(duration, callback);
 			t.Start();
+			return t;
```

Comment:
If you make this interface return `IDisposable` instead, this should IMO be much better.

This change reveal that there's a design-problem in this class as-is, though: there isn't anything explicitly keeping the `Timer`-instance alive. Yeah, there will probably be an implicit reference held because of the UpdateManager-subscription, but that feels pretty fragile.